apps/docs/content/components/Stack/index.mdx

```diff
@@ -0,0 +1,88 @@
+---
+title: Stack
+description: Use the stack component to enable layout of its immediate children along the vertical or horizontal axis.
+---
+
+import CustomVideoPlayer from '../../../src/components/custom-video-player'
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+![](https://github.com/primer/brand/assets/912236/f04a87ce-a4ca-403b-8732-a8575a39437a)
+
+## Usage
+
+Use the stack component to offer space distribution and alignment between items in a vertical or horizontal direction (one-dimensional layout)[^1]. If you need vertical and horizontal layout (two-dimensional layout), use the [grid](/components/Grid) component instead. When in doubt, we recommend reding the [grid vs stack](/components/Grid#grid-vs-stack) documentation.
+
+We recommend using the stack component for inline or text based elements. The stack allows scaling, wraping and shrinking features depending on each item content. For example, a navigational list or a button group.
```

Comment:
```suggestion
We recommend using the stack component for inline or text based elements. Stack enables items to scale, wrap, and shrink depending on each item's content and the available space.

Reply:
Removed the example here because it's repeated on the Do/Don't section below.

Reply:
👍 Also in case that suggestion isn't applied, there's a tiny typo on `wrapping`